connect/mirror-client/src/main/java/org/apache/kafka/connect/mirror/ReplicationPolicy.java

```diff
@@ -57,4 +57,9 @@ default boolean isInternalTopic(String topic) {
         return topic.endsWith(".internal") || topic.endsWith("-internal") || topic.startsWith("__")
             || topic.startsWith(".");
     }
+
+    /** Checks if the policy can track back to the source of the topic. */
+    default boolean canTrackSource(String topic) {
```

Comment:
If a public API change like this is required, you will need to propose a small KIP. I'm unclear why it's required tho, and ideally we would not alter the existing API if possible.

If a new method is required, I think "track" is too ambiguous and should not be used here.